Automattic/wp-calypso
# Performance Profiler: use fade in and out animation for tips

Related to https://github.com/Automattic/dotcom-forge/issues/9108
Follow-up to https://github.com/Automattic/wp-calypso/pull/95401

## Proposed Changes

* Update the tip update animation

## Why are these changes being made?


* Discussion on https://github.com/Automattic/wp-calypso/pull/95401

## Testing Instructions



* Go to `/speed-test` and start a new test
* Check that the tips on the loading screen are using a fade in and fade out animation when updated

**Before**

https://github.com/user-attachments/assets/afb87d8a-fe95-4c3e-9812-8ecfe3fc84c6

**After**

https://github.com/user-attachments/assets/4ee857c4-0506-413b-ae63-402839ec23c3
